### DISCUSSION TITLE
please add link to channels in chile

### DISCUSSION THREAD

**Discussion by olivertvitv** [iptv-org/community#1019] [Q&A]
Hello, please add link to the following Chilean channels
TV SALUD:  https://panel.miplay.cl:8081/tvsalud/index.m3u8
LOGO TV SALUD: https://mastermedia.cl/wp-content/uploads/2023/08/1-TV-SALUD-CHILE.png

SELLO DE RAZA : https://stmv5.voxtvhd.com.br/selloderazatv/selloderazatv/playlist.m3u8
LOGO SELLO DE RAZA: https://mastermedia.cl/wp-content/uploads/2024/01/icono.png

CHILE TRADICIONES TV: https://stmv5.voxtvhd.com.br/chiletradicionestv/chiletradicionestv/playlist.m3u8
LOGO CHILE TRADICIONES TV: https://mastermedia.cl/wp-content/uploads/2023/12/chiletradiciones01.png

THANK YOU SO MUCH

**Comment by freearhey** [comment]
All the links are dead.

![image](https://github.com/user-attachments/assets/6fdf932b-c6f9-4181-b8ed-4c65595ec308)
